<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PackageStylerDialog</class>
 <widget class="QDialog" name="PackageStylerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>150</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PackageStyler</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="configGroupBox">
     <property name="title">
      <string>Configuration</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="keywordLabel">
          <property name="text">
           <string>Keyword:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="keywordLineEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="qmlLabel">
          <property name="text">
           <string>QML File:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="qmlPathLineEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="browseButton">
          <property name="text">
           <string>Browse</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="addButton">
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="removeButton">
          <property name="text">
           <string>Remove</string>
          </property>
         </widget>
        </item>
      </layout>
      </item>
      <item>
       <widget class="QListWidget" name="configListWidget"/>
      </item>
      <item>
       <widget class="QPushButton" name="saveConfigButton">
        <property name="text">
         <string>Save Configuration</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <widget class="QPushButton" name="configureButton">
       <property name="text">
        <string>Configure Plugin</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="runButton">
       <property name="text">
        <string>Run</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveLayerButton">
       <property name="text">
        <string>Save Layer in Database</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
